test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -607,7 +607,7 @@ public void startup(ICluster cluster)
                 registerInboundFilter(cluster);
                 registerOutboundFilter(cluster);
 
-                JVMStabilityInspector.replaceKiller(new InstanceKiller());
+                JVMStabilityInspector.replaceKiller(new InstanceKiller(Instance.this::shutdown));
```

Comment:
added this so we actually shutdown the instance and not just throw the exception (still do that)